common/src/main/java/com/wynntils/mc/mixin/CapeLayerMixin.java

```diff
@@ -40,4 +47,28 @@ private void render(
             ci.cancel();
         }
     }
+
+    @ModifyArg(
```

Comment:
Does ModifyArg work on Neoforge? IIrc either `ModifyArg` or `ModifyArgs` did not use to work with Forge. I think the latter, as it caused some module generation - dependency resolution issues. But please do confirm this for me. Thanks.